test/D2L.Security.OAuth2.UnitTests/Keys/Default/RemotePublicKeyProviderTests.cs

```diff
@@ -17,14 +17,14 @@ internal sealed class RemotePublicKeyProviderTests {
 		private IPublicKeyProvider m_publicKeyProvider;
 
 		private string SRC_NAMESPACE;
-		private Guid KEY_ID;
+		private string KEY_ID;
 
 		[SetUp]
 		public void BeforeEach() {
 			m_jwksProvider = new Mock<IJwksProvider>( MockBehavior.Strict );
 			m_keyCache = new Mock<IInMemoryPublicKeyCache>( MockBehavior.Strict );
 			SRC_NAMESPACE = Guid.NewGuid().ToString();
-			KEY_ID = Guid.NewGuid();
+			KEY_ID = Guid.NewGuid().ToString();

```

Comment:
Somewhere along the line, it would be good to test non-guid key ids.

Reply:
- [X] tests with string KeyIds